frontend/src/metabase/ui/components/overlays/Modal/Modal.styled.tsx

```diff
@@ -16,6 +16,13 @@ export const getModalOverrides = (): MantineThemeOverride["components"] => ({
         backgroundColor: theme.fn.rgba(theme.fn.themeColor("bg-black"), 0.6),
       },
     }),
+    variants: {
+      "large-overlay-padding": () => ({
```

Comment:
I'd pick a semantic name if possible. Not sure if there is one. Someone from the design team could help

Reply:
agreed, we need a better name for the variant